server/segment_rpc.go

```diff
@@ -634,6 +635,15 @@ func genPayment(sess *BroadcastSession, numTickets int) (string, error) {
 		}
 
 		protoPayment.TicketSenderParams = senderParams
+
+		ratPrice, _ := ratPriceInfo(protoPayment.ExpectedPrice)
+		glog.V(common.VERBOSE).Infof("Created new payment - recipient=%v  faceValue=%v  winProb=%v  price=%v  numTickets=%v",
```

Comment:
Since we include the manifestID when logging on the orchestrator think it could be good to include the manifestID here as well

Reply:
fixed in 1a02391